src/tool/cpp/strings/base_activation.h

```diff
@@ -192,10 +199,6 @@ namespace winrt::impl
         template <typename F>
         auto call(F&& callback)
         {
-#ifdef WINRT_DIAGNOSTICS
-            get_diagnostics_info().add_factory<Class>();
-#endif
-
             {
```

Comment:
this was removed, but diagnostics_info is still around?

Reply:
Removed it all.

Reply:
it was still in base.h - was that generated before removal?

Reply:
Please check again. I don't see it in the PR.